src/audio/drc/drc_math_lib/asin/c_fixpt/drc_asin.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * File: drc_asin.h
+ *
+ * Author :Shriram Shastry
+ * Copyright (c) 2016, Intel Corporation All rights reserved.
+ */
+
+#ifndef DRC_ASIN_H
+#define DRC_ASIN_H
+
+/* Include Files */
+#include <stddef.h>
+#include <stdlib.h>
+#include "typedef.h"
+#include "drc_asin_types.h"
+
+/* Function Declarations */
+extern void drc_asin_fixpt(const signed char x[2], int y[2]);
```

Comment:
There should be a clear instruction comment to tell user what are the inputs and outputs, and their word length and fraction length, expected input and output value range, and etc.

For example, https://github.com/thesofproject/sof/blob/master/src/math/decibels.c#L74

Reply:
All other c-fixpt implementations as well.

Reply:
OK - I'II add for all library function